File: Services/Dto/PartnerDto.cs
using System;
using System.Collections.Generic;
using Services.Dto.CollectsDelivers;
using Services.Dto.Outsourced;
using Services.Dto.Shared;

namespace Services.Dto
{
    public class PartnerDto
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public DateTime Registered { get; set; }
        public string CNPJ { get; set; }
        public string Responsible { get; set; }
        public string Comments { get; set; }
        public string Businessline { get; set; }
        public bool Transporter { get; set; }
        public bool HardwareSupplier { get; set; }
        public bool EletronicRepair { get; set; }
        public AddressDto Address { get; set; }
        public ContactDto Contact { get; set; }
        public virtual List<CollectDeliverDto> CollectsDelivers { get; set; }
        public virtual List<EletronicRepairDto> EletronicsRepairs { get; set; }
    }
}

File: Views/src/shared/components/table-g/dtos/partner-dto.ts
import { AddressDto } from "./address-dto";
import { ContactDto } from "./contact-dto";
import { InventoryDto } from "./inventory-dto";
import { CollectDeliverDto } from "./collect-deliver-dto";

export class PartnerDto {
   id:number;
   name:string;
   today:Date;
   cnpj:string;
   responsible:string;
   comments:string;
   businessline:string;
   transporter:boolean;
   supplier:boolean;
   address:AddressDto;
   addressid:number;
   contact:ContactDto;
   contactid:number;
   inventories:InventoryDto[];
   transportercollectdelivers:CollectDeliverDto[];
   sourcecollectdelivers:CollectDeliverDto[];
   destinycollectdelivers:CollectDeliverDto[];



}
